lib/broccoli-style-export.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-env node */
+
+'use strict';
+
+const Filter = require('broccoli-persistent-filter');
+const extend = require('deep-extend');
+const path = require('path');
+const postcss = require('postcss');
```

Comment:
We are using `postcss` and `autoprefixer` instead of `broccoli-autoprefixer`, because the last does not filter file types (css) to pass autoprefixer, so It tries to prefix everything including javascript files, with results in an error.
To use `broccoli-autoprefixer` we have to create two separate funnels: one for styles and other for scripts, so I think this approach is simpler and probably faster.